README_validations.md

```diff
@@ -0,0 +1,129 @@
+# How to Validate OSCAL
+
+## XML or JSON?
+
+XML files use XML syntax: they are *well-formed* by the rules of XML. (See page on wf vs/and valid.) If you can process your file using a conformant XML processor, it is XML.
+
+Same for JSON (although of course you need a JSON processor, not an XML processor).
+
+But *validating* goes beyond parsing: it tells you not only that a file is can be read successfully, but also that its data structures conform to rules known ahead or stipulated for a particular application of XML or JSON. All XML is well-formed XML, but only OSCAL XML is valid to the OSCAL models. Likewise with JSON and YAML.
```

Comment:
```suggestion
But *validating* goes beyond parsing: it tells you not only that a file can be read successfully, but also that its data structures conform to rules for a particular application of XML, JSON, or YAML. Just because an XML instance is well-formed XML, doesn't mean it is valid OSCAL XML. Likewise with JSON and YAML.
```